community/nfs-utils/build

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --sysconfdir=/etc \
+    --disable-nfsv4 \
+    --disable-gss
+
```

Comment:
Possibly try:

Possibly try (right before `make`):

```sh
sed -i 's|sbin|usr/bin|g' Makefile
sed -i "s|sbin|bin|" utils/nfsidmap/id_resolver.conf
```

Also add `--sbindir=/usr/bin` to configure. :+1:

Let me know what the manifest looks like afterwards.